### DISCUSSION TITLE
Boot TWRP

### DISCUSSION THREAD

**Discussion by kis80gezenguz** [SebaUbuntu/TWRP-device-tree-generator#103] [Q&A]
Hi,

based on the wiki I successfully build TWRP for my device (Odys TIGERTAB), but unfortunately TWRP is not starting. 
Could you please advise me how to finde the problem.

**Comment by SebaUbuntu** [comment]
One suggestion is to check a device tree of a device with the same SoC (or a similar one) and find the differences
